GTHalfSheet/GTSheet/UI/DismissBar.storyboard

```diff
@@ -30,9 +28,9 @@
                                     </userDefinedRuntimeAttribute>
                                 </userDefinedRuntimeAttributes>
                             </view>
-                            <view alpha="0.25" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="oez-Fr-Vle">
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="oez-Fr-Vle">
                                 <rect key="frame" x="165.5" y="661" width="44" height="6"/>
-                                <color key="backgroundColor" white="0.0" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <color key="backgroundColor" systemColor="tertiaryLabelColor" red="0.23529411759999999" green="0.23529411759999999" blue="0.26274509800000001" alpha="0.29999999999999999" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
`tertiaryLabelColor` only works in > iOS 13. What happens when run against iOS 12? I'm not sure if Storyboards take care of this for automatically or not